### PR TITLE
Refined Release tagging

### PR DESCRIPTION
**Pull request by UtkarshBhatthere**
Once this PR is merged, we need to test the tagging by git tagging the any commit as `v17.2.6-beta`. The expected tags are:
1. v17.2.6-beta
2. v17.2.6-deb-ver
3. quincy

### CODE REVIEW

File: .github/workflows/publish_release.yaml [hunk 1]
```diff
@@ -44,7 +37,7 @@ jobs:
           DEB_VER=$(cut -d'-' -f2 <<< $PKG_VER)
```
Comment (sabaini) [inline .github/workflows/publish_release.yaml:37]: Afaict we don't need the $DEB_VER var anymore, right? If so could remove it

**Review by sabaini — CHANGES_REQUESTED** [review]
Thanks Utkarsh, much appreciated. One minor cleanup request, otherwise lgtm

File: .github/workflows/publish_release.yaml (hunk 1 above)
Comment (UtkarshBhatthere) [reply .github/workflows/publish_release.yaml]: removed.

**Review by sabaini — APPROVED** [review]

**Comment by UtkarshBhatthere** [comment]
image
The released tags match with the expectations mentioned in the PR above.

**Pull request merged**